user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentalGroupInput.java

```diff
@@ -79,12 +83,18 @@ public ExperimentalGroupInput(Collection<VariableLevel> availableLevels, boolean
         event -> setInvalid(variableLevelSelect.isInvalid() || replicateCountField.isInvalid()));
     replicateCountField.addValueChangeListener(
         event -> setInvalid(variableLevelSelect.isInvalid() || replicateCountField.isInvalid()));
+    nameField.addValueChangeListener(
+        event -> setInvalid(variableLevelSelect.isInvalid() || replicateCountField.isInvalid()));
```

Comment:
I don't think this is necessary here, since there is no value that would set the other fields as valid.